src/components/shared/card.tsx

```diff
@@ -268,6 +292,7 @@ const iconStyles = (format: Format): SerializedStyles => {
 		display: inline-block;
 		background-color: ${inverted};
 		border-radius: 50%;
+		margin-top: -4px;
```

Comment:
Possibly `- remSpace[1]`? I realise SVG alignment is tricky so 👍 if this doesn't work.